engines/grim/detection.cpp

```diff
@@ -56,6 +56,15 @@ static const ADExtraGuiOptionsMap gameGuiOptions[] = {
 			false
 		}
 	},
+	{
+		GAMEOPTION_LOAD_DATAUSR,
```

Comment:
You'll have to define another name for your new option, and use it in the detection entries.